tests/konsist/src/test/kotlin/io/element/android/tests/konsist/KonsistArchitectureTest.kt

```diff
@@ -55,4 +62,33 @@ class KonsistArchitectureTest {
             .withNameEndingWith("Events")
             .assertEmpty(additionalMessage = "Events class MUST be sealed interface")
     }
+
+    @Test
+    fun `Sealed class without constructor and without parent MUST be sealed interface`() {
+        Konsist.scopeFromProject()
+            .classes()
+            .withSealedModifier()
+            .withoutConstructors()
+            .withoutParents()
+            .assertEmpty(additionalMessage = "Sealed class without constructor MUST be sealed interface")
+    }
+
+    @Test
+    fun `Composable MUST not have sealed interface in parameter`() {
```

Comment:
Maybe change this to `Sealed interface used in Composable MUST be Immutable or Stable`?

Reply:
Yes, you're right, I forgot to rename the test: [07fc0bd](https://github.com/vector-im/element-x-android/pull/1731/commits/07fc0bde0fce9dd935c10e4da9802848bfd964f5)